pyp2rpm/bin.py

```diff
@@ -246,7 +246,13 @@ def main(package, v, prerelease, d, s, r, proxy, srpm, p, b, o, t, venv, autonc,
 
         if srpm:
             msg = utils.build_srpm(spec_path, d)
-            logger.info(msg)
+            if isinstance(msg, bytes):
+                for line in msg.decode().split('\n'):
```

Comment:
This should be more robust:

```suggestion
                for line in msg.decode('utf-8', errors='replace').split('\n'):
```

Reply:
Good point.